lib/ApiClient.php

```diff
@@ -96,6 +96,15 @@ public function authorize(array $params) {
         return $this->httpClient->post($this->getUrl('merchant/transactions/authorize'), $params);
     }
 
+    /**
+     * Manually review a transaction that is in the reversibly_rejected state.
+     * @param array $params
+     * @return Response
+     */
+    public function review(array $params) {
+        return $this->httpClient->post($this->getUrl('merchant/transactions/review'), $params);
+    }
```

Comment:
should we have tests for these functions to make sure that they're returning the correct thing?

Reply:
@kajan-bolt We have tests for the post endpoint and all endpoints share the same HTTP header, so I think we are good.

Reply:
sounds good